docs/ci-cd.md

```diff
@@ -1,3 +1,34 @@
 # CI/CD
 
-TODO
+### GitLab CI/CD
+
+Set the following variables in GitLab project:
+
+- `SSH_PRIVATE_KEY`: Content of `~/.ssh/known_hosts` file. 
+The public SSH keys for a host may be obtained using the utility `ssh-keyscan`. 
+For example: `ssh-keyscan deployer.org`.
+- `SSH_KNOW_HOSTS`: Private key for connecting to remote hosts. 
+To generate private key: `ssh-keygen -t ed25519 -C 'gitlab@deployer.org'`.
+
+Create .gitlab-ci.yml file with following content:
+
+```yml
+stages:
```

Comment:
Also, is there a way to prevent concurrent deploys at the same time?

For example, if two commits was made at the same time to master?

Reply:
Done, there is a specific section in the [GitLab docs](https://docs.gitlab.com/ee/ci/environments/deployment_safety.html).